manifests/s/Sunnystew/SpaceShooter/1.0.0.0/Sunnystew.SpaceShooter.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Sunnystew.SpaceShooter
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: Sunnystew
+PackageName: Space Shooter
+License: Proprietary
+Copyright: Copyright © Sunnystew - All rights belong to Sunnystew - Some assets and sound effects have been used from Kenney.nl
+ShortDescription: Installer for Space Shooter

```

Comment:
Please provide a better description of what the application is, does, or is used for